src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="logo" href="#home">
+          <img src="./images/logo.png" alt="logo moyo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li><a href="#" class="category is-active">Apple</a></li>
+            <li><a href="#" class="category">Samsung</a></li>
+            <li><a href="#" class="category">Smartphones</a></li>
```

Comment:
Continue use BEM namig
```suggestion
          <ul class="nav__list">
            <li><a href="#" class="nav__link is-active">Apple</a></li>
            <li><a href="#" class="nav__link">Samsung</a></li>
            <li><a href="#" class="nav__link">Smartphones</a></li>
```